contracts/mixnet/src/contract.rs

```diff
@@ -24,6 +24,7 @@ pub const INITIAL_MIXNODE_BOND: Uint128 = Uint128(100_000000);
 pub const INITIAL_MIXNODE_BOND_REWARD_RATE: u64 = 110;
 pub const INITIAL_MIXNODE_DELEGATION_REWARD_RATE: u64 = 110;
 
+pub const INITIAL_MIXNODE_DEMANDED_SET_SIZE: u32 = 200;
```

Comment:
Wasn't there a discussion that we should have a big number here, so that all the current bonded nodes in the testnet receive something?

Reply:
That's a good question, but realistically when we deploy it, it will be on the new testnet where I think (?) we will want to have a more realistic number here, i.e. in the order of hundreds. But I might be wrong